pt.html

```diff
@@ -201,96 +207,97 @@
+	de aulas de francês no ensino médio.
 </words>
 </panel>
 
 <panel w=500 h=400>
 <pic src="pics/intro0.png" sx=950 sy=450></pic>
 <words x=10 y=10 w=400 h=60>
-	Since then, I've used Spaced Repetition to remember all sorts of things...
+	Desde então, passei a usar a Repetição Espaçada para lembrar de todo tipo de coisa...
 </words>
 <words x=81 y=113 w=100 style="text-align:left" no-bg class="comic_text">
-	ukulele chords
+	acordes de ukulele
 </words>
 <words x=212 y=114 w=100 style="text-align:left" no-bg class="comic_text">
-	computer code
+	código de programação
 </words>
 <words x=380 y=137 w=100 style="text-align:left" no-bg class="comic_text">
-	friends' birthdays
+	aniversário de amigos
 </words>
 <words x=188 y=204 w=270 style="text-align:left" no-bg class="comic_text">
-	anything interesting i find in books, talks, articles, etc!
+	qualquer coisa interessante que eu encontre em livros, palestras, artigos, etc!
 </words>
 <words x=60 y=300 w=400 h=60>
-	...and this lil' memory card game became a core part of my <i>life</i>.
+	...e este pequeno jogo de cartas de memória tornou-se uma parte essencial da minha <i>vida</i>.
 </words>
 </panel>
 
 <panel w=550 h=350>
 <pic src="pics/intro0.png" sx=0 sy=850></pic>
 <words x=10 y=10 w=500 h=30>
-	In short, Spaced Repetition = testing + time.
+	Resumidamente, Repetição Espaçada = ensaio + tempo
 </words>
 <words x=10 y=220 w=500 h=90>
-	You test yourself on a fact repeatedly, spacing out your repetitions over time.
-	(But won't this take forever? Ah, as we'll see later, there's a trick...)
+	Você se testa em um fato repetidamente, espaçando suas repetições ao longo do tempo.
+	(Mas isso não vai demorar para sempre? Ah, como veremos mais tarde, há um truque...)
 </words>
 </panel>
 
 <panel w=400 h=400>
 <pic src="pics/intro0.png" sx=0 sy=1200></pic>
 <words x=10 y=20 w=290 no-bg>
-	Spaced Repetition is free, evidence-based, and so simple you can do it with a <i>shoebox</i>.
+	Repetição Espaçada é gratuita, baseada em evidências, e tão simples que você pode fazê-lo com uma <i>caixa de sapatos</i>.
 </words>
 <words x=308 y=265 w=50 no-bg class="comic_text" style="text-align:left; font-size:20px">
-	nicky shut up
+	nicky cala boca
 </words>
 <words x=90 y=130 w=270 no-bg>
-	So, what's the catch? Why isn't <i>everyone</i> already doing Spaced Repetition?
+	Então, qual é o truque? Por que nem <i>todo mundo</i> já está fazendo Repetição Espaçada?
 </words>
 </panel>
 
 <panel w=400 h=400>
 <pic src="pics/intro0.png" sx=400 sy=1200></pic>
 <words x=10 y=20 w=330 no-bg>
-	Well, the catch is that making <i>any</i> new habit is hard
-	– especially a weird habit like Spaced Repetition.
+	Bem, a pegadinha é que fazer <i>qualquer</i> novo hábito é difícil
+	- especialmente um hábito estranho como Repetição Espaçada.
 </words>
 <words x=292 y=303 w=80 no-bg class="comic_text" style="text-align:left; font-size:20px">
-	teacher's coming
+	professor está vindo
```

Comment:
Acho que dá para mudar aqui e deixar mais informal

```suggestion
	professor tá vindo
```

ou até

```suggestion
	sor tá vindo
```

😛

Reply:
Esse aqui dependendo deixa marcado com estrelinha pra discutir mais tarde.

Ate pensando em revisar isso com contatos meus que são de outros países que falam português, mas o dialeto deles não é pt-BR